src/analytics/selectors.ts

```diff
@@ -90,10 +89,6 @@ export const getCurrentUserTraits = createSelector(
       localCurrencyCode,
       hasVerifiedNumber,
       hasCompletedBackup,
-      deviceId: DeviceInfo.getUniqueId(),
-      appVersion: DeviceInfo.getVersion(),
-      appBuildNumber: DeviceInfo.getBuildNumber(),
-      appBundleId: DeviceInfo.getBundleId(),
```

Comment:
I think we should have kept these.

Reply:
how come? is it because of the saga that calls the `identify` method on the analytics when these things change? i thought that these variables were unlikely to change during a session so seemed like a safe change?

Reply:
I commented here: https://github.com/valora-inc/wallet/pull/3294

Reply:
yeah it's annoying that they're in every track event in BQ twice, but that doesn't mean it's safe to remove them from one of the places yet imo

Reply:
also if we're going to remove them i'm not sure i'd do it from here